apps/meteor/server/services/video-conference/service.ts

```diff
@@ -1097,8 +1097,15 @@ export class VideoConfService extends ServiceClassInternal implements IVideoConf
 			return;
 		}
 
-		const name = settings.get<string>('VideoConf_Persistent_Chat_Discussion_Name') || i18n.t('Conference Call Chat History');
-		const displayName = `${name} - ${new Date().toISOString().substring(0, 10)}`;
+		const name = settings.get<string>('VideoConf_Persistent_Chat_Discussion_Name') || i18n.t('[date] Video Call Chat');
```

Comment:
Doesn't i18n allows to pass params? wonder if it would be better to use i18n compatible placeholders to avoid doing it manually 👀 

or was it a requirement to have it in this format?

Reply:
The actual value of the setting is not a i18n string, just a normal string that the user inputs, so no i18n functions available

Reply:
Yeah, I mean, can't we treat it as an i18n string? 

if it's not possible, pls ignore this one